examples/navier-stokes/nsplex.c

```diff
@@ -722,13 +741,19 @@ int main(int argc, char **argv) {
   {  // Print grid information
     CeedInt gnodes, onodes;
     int comm_size;
+    char box_faces_str[PETSC_MAX_PATH_LEN];
```

Comment:
Is there a better size to use here other than `PETSC_MAX_PATH_LEN`?

Reply:
I think this should be fine.